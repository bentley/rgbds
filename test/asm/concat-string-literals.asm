	println "Hello," /* comment */ " " \
		"world""!"

two EQUS "\"2\""

	println "4^" two " = 16"

	println """multi-
line"""" """"con\ ; comment
cat""" " works"

makeprint: MACRO
\1 EQUS "println " \2
ENDM
	makeprint printfour, {two}"+"{two}
	printfour

	println "literal" STRCAT("expr") ; error
