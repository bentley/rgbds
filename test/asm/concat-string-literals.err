ERROR: concat-string-literals.asm(17):
    syntax error
error: Assembly aborted (1 errors)!
