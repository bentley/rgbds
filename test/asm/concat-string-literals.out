Hello, world!
4^2 = 16
multi-
line concat works
$4
